webplugin/js/app/km-rich-text-event-handler.js

```diff
@@ -292,18 +292,16 @@ Kommunicate.richMsgEventHandler = {
             data[inputs[i].name] = inputs[i].value; 
         }
         if (requestType == "json") {  
-           KommunicateUtils.isURL(form.action) && window.Applozic.ALApiService.ajax({
```

Comment:
@ab14bhardwaj 
Some API calls are not happening due to CORS restriction, this error is happening with Applozic.ALApiService.ajax() method. after replacing that method with javascript XMLHttpRequest() POST request is worked. So do we have any CROS restrictions at chat-widget?

Reply:
in the common ajax method, we are adding our own request headers which our servers require. for external calls, we can avoid using the common code.